src/main/java/uk/gov/hmcts/darts/common/entity/EventEntity.java

```diff
@@ -35,7 +36,7 @@ public class EventEntity extends VersionedEntity {
     @Column(name = "event_object_id", unique = true, length = 16)
     private String legacyObjectId;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.EAGER, optional = false)
     @JoinColumn(name = "evh_id")
     private EventHandlerEntity eventType;
```

Comment:
Since we're touching this relationship - would you be able to correct the field please so that it represents the new entity name - `eventHandler`?

Reply:
This requires quite a few changes so can be moved to another ticket